mala/datahandling/data_scaler.py

```diff
@@ -295,6 +316,8 @@ def transform(self, unscaled):
 
         # Perform the actual scaling, but use no_grad to make sure
         # that the next couple of iterations stay untracked.
+        scaled = unscaled if copy is False else unscaled.clone()
```

Comment:
```suggestion
        scaled = unscaled.clone() if copy else unscaled
```
is maybe a bit more pythonic

Reply:
Sounds good, thanks for the suggestion!